apps/www/content/docs/react-19.mdx

```diff
@@ -136,7 +136,7 @@ To make it easy for you track the progress of the upgrade, I've created a table
 
 | Package                                                                            | Status | Note                                                        |
 | ---------------------------------------------------------------------------------- | ------ | ----------------------------------------------------------- |
-| [radix-ui](https://www.npmjs.com/package/@radix-ui/react-icons)                    | ✅     |                                                             |
+| [radix-ui](https://www.npmjs.com/package/radix-ui)                                 | ✅     |                                                             |
```

Comment:
This is not correct. The [`radix-ui`](https://www.npmjs.com/package/radix-ui) package doesn't include Radix Icons.